src/home/components/HomePageView.tsx

```diff
@@ -37,9 +39,14 @@ const HomePageView = (): JSX.Element => {
     return () => removeListeningGameCreated();
   }, []);
 
-  const handleCreateGame = useCallback(() => {
-    if (player) emitCreateGame(player);
-  }, [player]);
+  /**
+   * Handle create game.
+   *
+   * @return {void}
+   */
+  const handleCreateGame = (): void => {
+    if (player) emitCreateGame({ player, settings: gameSettings ?? undefined });
```

Comment:
Unless I am mistaken, something like this should work
```suggestion
    if (player) {
      emitCreateGame({ 
        player,
        (...gameSettings && {settings: gameSettings}) 
      })
    }
```

Reply:
Hmmm ça ne passe pas :

![image](https://github.com/KinderCorp/battleship-front/assets/46479135/cd61797b-5eed-4371-b864-486f236ad061)

Je ne sais pas d'ailleurs s'il existe un opérateur spécifique en JS pour faire ça...

Reply:
:heavy_check_mark: